test/shim_exec.bats

```diff
@@ -116,9 +116,7 @@ teardown() {
 @test "shim exec should execute first plugin that is installed and set" {
   run asdf install dummy 3.0
 
-  echo "dummy 1.0" > $PROJECT_DIR/.tool-versions
-  echo "dummy 3.0" >> $PROJECT_DIR/.tool-versions
-  echo "dummy 2.0" >> $PROJECT_DIR/.tool-versions
```

Comment:
What is the expected behavior for scenarios like this? Even though this isn't supported it might be good to have a test that shows only the first version is used and the others are ignored.

Reply:
Sounds good, I'll add a test for this and merge.